Sources/OktaOidc/Common/OktaOidcConfig.swift

```diff
@@ -30,7 +30,9 @@ public class OktaOidcConfig: NSObject {
      requests throughout OktaOidc. More information could be found here: https://github.com/okta/okta-oidc-ios/blob/master/README.md#modify-network-requests.
      */
     @objc public weak var requestCustomizationDelegate: OktaNetworkRequestCustomizationDelegate?
-
+    
+    @objc public weak var oktaCustomTokenValidator: OktaCustomTokenValidator?
```

Comment:
Why `weak`? That means a user should keep strong reference somewhere to keep it alive. 
I would suggest make it strong reference, then a user can set it in one line: `config.oktaCustomTokenValidator = Validator()`